### ISSUE TITLE
Add capabilities

### ISSUE THREAD

**Issue by laurentkempe** [MicrosoftDocs/msteams-docs#3559]
Hi,

The doc states as the step 5 of [Set up a new Teams project](https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/visual-studio-overview#set-up-a-new-teams-project)

    A pop-up window labeled Add Capabilities will allow you to choose one or more capabilities for your project setup.

I installed Teams Toolkit 0.1.44.25044 for Visual Studio and I don't see such a popup.

How can I add bot capability to the project?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d7a0c9e9-d1d1-b412-4d6c-a7f16237c8ab
* Version Independent ID: b4960863-6765-b6f2-3fd8-c9316f2cb2e8
* Content: [Build apps with the Microsoft Teams Toolkit and Visual Studio - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/visual-studio-overview)
* Content Source: [msteams-platform/toolkit/visual-studio-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/toolkit/visual-studio-overview.md)
* Product: **msteams**
* GitHub Login: @laujan
* Microsoft Alias: **lajanuar**

**Comment by Sridevi-MSFT** [comment]
@laurentkempe - I have tried to set up a new Teams project following the steps provided in documentation with Teams Toolkit version 0.1.44.25044 and able to see a pop up to Add Capabilities. So, after step#4 click on **create** button to get  Add Capabilities popup.

**Comment by laurentkempe** [comment]
@Sridevi-MSFT This is the experience I am having! I am on Microsoft Visual Studio Professional 2019
Version 16.10.0. I run Repair of VS from the installer and I still have the same problem, I don't see the pop-up window labeled Add Capabilities.

https://user-images.githubusercontent.com/272612/120786718-d8b77000-c52e-11eb-86cd-c97135020a28.mp4

**Comment by Sridevi-MSFT** [comment]
@laurentkempe - Seems like you are not using teams toolkit to create a new Teams Project. While creating new project please select Microsoft Teams App with quickstart tool as shown in below image. 

![image](https://user-images.githubusercontent.com/80752198/120888239-d4658280-c614-11eb-9f06-fedd78be9481.png)

**Comment by laurentkempe** [comment]
Thanks @Sridevi-MSFT 
As I wrote on #3560, it is [hard to follow the instructions](https://github.com/MicrosoftDocs/msteams-docs/issues/3560#issuecomment-854822697)
